### ISSUE TITLE
Error in genmsg when generating the Empty.srv service

### ISSUE THREAD

**Issue by wjwwood** [ros/genmsg#6]
When building the base stacks with catkin/groovy/OS X 10.7.x I get:

```
∫ make
[  0%] Building CXX object rospack/CMakeFiles/rospack.dir/src/rospack.cpp.o
[  0%] Building CXX object rospack/CMakeFiles/rospack.dir/src/rospack_backcompat.cpp.o
[  0%] Building CXX object rospack/CMakeFiles/rospack.dir/src/utils.cpp.o
[  1%] Building CXX object rospack/CMakeFiles/rospack.dir/src/rospack_cmdline.cpp.o
[  1%] Building CXX object rospack/CMakeFiles/rospack.dir/tinyxml-2.5.3/tinyxml.cpp.o
[  1%] Building CXX object rospack/CMakeFiles/rospack.dir/tinyxml-2.5.3/tinyxmlparser.cpp.o
[  1%] Building CXX object rospack/CMakeFiles/rospack.dir/tinyxml-2.5.3/tinyxmlerror.cpp.o
Linking CXX shared library ../buildspace/lib/librospack.dylib
[  1%] Built target rospack
[  1%] Building CXX object rospack/CMakeFiles/rospackexe.dir/src/rospack_main.cpp.o
Linking CXX executable ../buildspace/bin/rospack
[  1%] Built target rospackexe
[  1%] Building CXX object rospack/CMakeFiles/rosstackexe.dir/src/rosstack_main.cpp.o
Linking CXX executable ../buildspace/bin/rosstack
[  1%] Built target rosstackexe
[  1%] Building CXX object ros/core/roslib/CMakeFiles/roslib.dir/src/package.cpp.o
Linking CXX shared library ../../../buildspace/lib/libroslib.dylib
[  1%] Built target roslib
[  1%] Building CXX object roscpp_core/cpp_common/CMakeFiles/cpp_common.dir/src/debug.cpp.o
Linking CXX shared library ../../buildspace/lib/libcpp_common.dylib
[  1%] Built target cpp_common
[  1%] Building CXX object roscpp_core/rostime/CMakeFiles/rostime.dir/src/time.cpp.o
[  1%] Building CXX object roscpp_core/rostime/CMakeFiles/rostime.dir/src/rate.cpp.o
[  1%] Building CXX object roscpp_core/rostime/CMakeFiles/rostime.dir/src/duration.cpp.o
Linking CXX shared library ../../buildspace/lib/librostime.dylib
[  1%] Built target rostime
[  1%] Building CXX object roscpp_core/roscpp_serialization/CMakeFiles/roscpp_serialization.dir/src/serialization.cpp.o
Linking CXX shared library ../../buildspace/lib/libroscpp_serialization.dylib
[  1%] Built target roscpp_serialization
[  1%] Generating C++ code from std_msgs/Bool.msg
[  1%] Generating C++ code from std_msgs/Byte.msg
[  1%] Generating C++ code from std_msgs/ByteMultiArray.msg


....


Scanning dependencies of target rosgraph_msgs_genpy
[ 37%] Generating Python from MSG rosgraph_msgs/Clock
[ 37%] Generating Python from MSG rosgraph_msgs/Log
[ 37%] Generating Python msg __init__.py for rosgraph_msgs
[ 37%] Built target rosgraph_msgs_genpy
[ 37%] Generating C++ code from std_srvs/Empty.srv
[ 37%] Built target std_srvs_gencpp
[ 37%] Generating Lisp code from std_srvs/Empty.srv
Traceback (most recent call last):
  File "/Users/william/devel/groovy/base/install/lib/python2.7/site-packages/genlisp-0.4.0-py2.7.egg/genlisp/genlisp_main.py", line 35, in genmain
    search_path = genmsg.command_line.includepath_to_dict(options.includepath)
  File "/Users/william/devel/groovy/base/install/lib/python2.7/site-packages/genmsg/command_line.py", line 35, in includepath_to_dict
    for path in includepath:
TypeError: 'NoneType' object is not iterable
ERROR:  'NoneType' object is not iterable
make[2]: *** [buildspace/share/common-lisp/ros/std_srvs/srv/Empty.lisp] Error 3
make[1]: *** [ros_comm/messages/std_srvs/CMakeFiles/std_srvs_genlisp.dir/all] Error 2
make: *** [all] Error 2
```

Not sure what brings this about, it looks like genlisp_main.py is getting called without the includepath command line option being set.  I don't know if this just needs to check for None and return a default value or if the real problem is that the includepath command line option is not getting set.

This problem does occur with plain old `make` as well.

**Comment by dirk-thomas** [comment]
iterable and command_line.py:35 looks like an outdated checkout of genmsg.
This should have been fixed in c677082c9ff0175d417b79c6eaf9c6b353f2a714

**Comment by wjwwood** [comment]
You're right, I had forgotten to clean my shell, it was pulling the file from my old install directory.

Works now.
